docs/src/sqlitegen/all.html

```diff
@@ -82,6 +82,8 @@ <h3>create-proc:</h3>
 <img src="create-proc.gif">
 <h3>create-trigger:</h3>
 <img src="create-trigger.gif">
+<h3>create-audit-trigger:</h3>
+<img src="create-audit-trigger.gif">
```

Comment:
But they aren't the same because you can't currently have more than one table for the audit trigger

Reply:
Given that I'm changing that fact I'll obviously fix this too

Reply:
Right, but we can handle it differently in the parser. Allow `AUDIT` optionally in the syntax and if you detect more that one table, abort with an appropriate error; something like "`AUDIT TRIGGER` currently does not support multiple tables", rather than a syntax error - which could be confusing.